bolt_support.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "BOLT protocol support"
+nav_order: 10
+description: "Connecting FalkorDB using BOLT protocol."
+---
+
+## BOLT protocol support for FalkorDB
+
+FalkorDB provides support for querying using BOLT drivers. This guide will walk you through the process of connecting to FalkorDB using the BOLT protocol.
+
+### Prerequisites
+
+Before you begin, ensure that you have a FalkorDB instance up and running. You can use either Docker or the FalkorDB sandbox for this purpose. Additionally, install the necessary BOLT drivers:
+
+```bash
+pip install neo4j
+```
+
+### Step 1: Create a `main.py` File
+
+Create a main.py file with the following content:
+
+```python
+from neo4j import GraphDatabase
+
+driver = GraphDatabase.driver("bolt://localhost:7687", auth=("falkordb", ""))
+
+records, _, _ = driver.execute_query(
+    "UNWIND range(1, $n) AS i RETURN i",
+    n=10, database_="mygraph",
+)
```

Comment:
except for `records` are the addition two discarded agreement are of any value ? (what are they) ?
I find it a bit annoying that the user need to specify the `database` for each query, moreover why is the parameter called `database_` and not `database` ?

Reply:
https://pypi.org/project/neo4j/ take a look at the first example

Reply:
they can just read the doc for this driver